tasks/main.yml

```diff
@@ -9,7 +9,7 @@
 - name: Fail if existing screen session
   fail:
     msg: A stress session is already running. It might be some pending writes to disk.
-  when: screen_files.matched != 0
+  when: screen_files.matched != 0 and skip_screen_check == True
```

Comment:
Maybe the variable should be "enable_screen_check" because if that variables is True and it finds files it will fail.
If the variable is False it will not?